crates/agglayer-node/Cargo.toml

```diff
@@ -17,8 +17,8 @@ hex.workspace = true
 hyper.workspace = true
 jsonrpsee = { workspace = true, features = ["full"] }
 parking_lot.workspace = true
-pin-project = "1.1.7"
-reqwest = "0.12.9"
+pin-project = "1.1.8"
```

Comment:
I'd say `pin-project` is a fairly generic dependency, could be moved to workspace deps.